internal/driver/driver.go

```diff
@@ -175,7 +172,7 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsModels.As
 		if err != nil {
 			d.lc.Error("Unsupported protocol mapping.",
 				"error", err,
-				"protocols", fmt.Sprintf("%v", device.Protocols),
+				"protocols", fmt.Sprintf("%+v", device.Protocols),
```

Comment:
`device.Protocols` is a `map`, so the `+` doesn't give us anything. (only useful for `struct` types).
```suggestion
				"protocols", fmt.Sprintf("%v", device.Protocols),
```

Reply:
removed +